src/raylib.h

```diff
@@ -852,6 +852,14 @@ typedef enum {
 // Callbacks to be implemented by users
 typedef void (*TraceLogCallback)(int logType, const char *text, va_list args);
 
+// Filesystem override implemented by users.
```

Comment:
Probably this filesystem related code shouldn't be in this PR.

Reply:
Yes, that's not excepted, it needs to be stripped out of this PR.